tests/test_data_sources.py

```
@@ -72,21 +76,21 @@ def test_opensky_datasource_stream(spark):
 def test_salesforce_datasource_registration(spark):
     """Test that Salesforce DataSource can be registered and validates required options."""
     spark.dataSource.register(SalesforceDataSource)
-    
+
     # Test that the datasource is registered with correct name
     assert SalesforceDataSource.name() == "salesforce"
```

Ruff (0.12.2)

`SalesforceDataSource` may be undefined, or defined from star imports

(F405)